bin/hdfcoinc/pycbc_multiifo_coinc_statmap

```diff
@@ -68,6 +68,10 @@ parser.add_argument('--hierarchical-removal-against', type=str,
                          'triggers that are louder than either the "inclusive"'
                          ' (little-dogs-in) background, or the "exclusive" '
                          '(little-dogs-out) background. [default="none"]')
+parser.add_argument('--additional-event-times', type=float, nargs="+",
+                    help="Additional event times which will be removed from "
```

Comment:
if you want several times should they be space- or comma-separated? maybe indicate this in the help message?